packages/design-system/src/components/popover.tsx

```diff
@@ -133,7 +133,7 @@ export const FloatingPopover = ({
           sideOffset={sideOffset}
           side="right"
           hideArrow
-          css={{ backgroundColor: "$loContrast" }}
+          css={{ backgroundColor: "$loContrast", minWidth: 240 }}
```

Comment:
I don't think this is the right place for this minWidth value, because you would be limiting any popover to this, which may not be true everywhere